packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -542,15 +544,25 @@ export default async function initSanity(
   // Bootstrap Sanity, creating required project files, manifests etc
   await bootstrapTemplate(templateOptions, context)
 
-  // Now for the slow part... installing dependencies
-  const pkgManager = await getPackageManagerChoice(outputPath, {
-    prompt,
-    interactive: unattended ? false : isInteractive,
-  })
+  let pkgManager: PackageManager
+
+  // If the user has specified a package manager, and it's allowed use that
+  if (packageManager && ALLOWED_PACKAGE_MANAGERS.includes(packageManager)) {
+    pkgManager = packageManager
+  } else {
```

Comment:
It looks like if the user specifies a disallowed package manager then we silently fall back to an allowed one. Should we warn the user?

Reply:
can, what would the copy look like?

Reply:
Something like this?

```ts
output.print(`Given package manager "${packageManager}" isn't supported. Supported package managers are ${allowedPms}.`)
output.print(`Continuing using ${pkgManager}...`)
```

Reply:
Does this make sense? 

![Screenshot 2024-05-31 at 10 12 26 AM](https://github.com/sanity-io/sanity/assets/6476108/6112d373-4c10-498d-b8ff-d7c7e1949384)

Reply:
That looks good to me!

(However, we should consider supporting `dog`).